Template/Template/Controllers\/OrderViewController.swift

```diff
@@ -4,8 +4,105 @@
 import UIKit
 
 /// экран с итоговой суммой
```

Comment:
```suggestion
/// экран с итоговой суммой
```
итоговой суммой чего?

Reply:
Заказа )